gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -136,6 +121,12 @@ def add_arguments(self, parser):
     parser.add_argument('--output_table',
                         default='',
                         help='BigQuery table to store the results.')
+    parser.add_argument(
```

Comment:
I'd really prefer we not include this flag yet.  If we specifically want to include functionality for lower cost backups (and the savings over compressed VCF are real) I'd rather introduce that as a feature than as an accidental feature like this.